### DISCUSSION TITLE
Compare multiple sources in main chart

### DISCUSSION THREAD

**Discussion by kleinicke** [plausible/analytics#980] [Anything Else]
Currently only one curve is visible in the main chart.
If would be interesting to see multiple sources/browser versions... in the main chart to compare the different performances over time. Currently it is only possible to set the filter to one selection and see the overall performance changes and afterwards switch to another filter.
With this filter a graphic like this:
https://commons.wikimedia.org/wiki/File:BrowserUsageShare.png
would be possible, just not limited on browsers.

**Comment by kleinicke** [comment]
A UI solution for this feature would be to put in into the details view of a single field.
![MultipleUIsuggestions](https://user-images.githubusercontent.com/10370975/116811496-01bc9d80-ab4a-11eb-8e39-dbaac3a2a695.jpg)

The image above could be an inspiration. When you click at "Compare in Main Chart" the selection boxes in front of the different sources and the "Show Combined" field show up. When one Source is selected, the "Show other" field shows up.
This selection is saved as a special filter that can be combined with other filters.

**Comment by ukutaht** [comment]
I've thought about making the 'Top Sources' report (when you click on 'details') be a more elaborate page on its own. It could show a big graph at the top comparing traffic over time from your top sources. And you could include/exclude stuff from the graph manually.

The tradeoff is making the product bigger/more complex. I really like the fact that everything fits on one page at the moment.

I'm not currently focused on this part of the app and it's not on our short-term roadmap. But I do agree that it would be great to have more charting options, especially for comparing traffic from multiple sources.
